ci/upload-bazel-metrics.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+steps:
+  - template: bash-lib.yml
+    parameters:
+      var_name: bash_lib
+  - bash: |
```

Comment:
Answering my own question: this takes less than a minute so not an issue for CI times.